pyrene/src/components/Modal/modal.css

```diff
@@ -9,15 +9,21 @@
   left: 0;
   right: 0;
   bottom: 0;
+  padding: 24px;
   background-color: rgba(29, 39, 59, 0.5);
   z-index: 2;
 
   display: flex;
   align-items: center;
   justify-content: center;
+  box-sizing: border-box;
```

Comment:
Don't we have box-sizing by default ?

`box-sizing: border-box;`

Reply:
It appears not to be the case, at least not within Kitchensink. Maybe the website that uses this component has set it globally (for example on `*`), but we can't depend on that I think.